.github/workflows/docker-build.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           load: true
           context: docker
           file: docker/Dockerfile
-          tags: jhudsl/course_template
+          tags: jhudsl/base_ottr
```

Comment:
Let's keep an eye on this - not sure what's new in the new docker

Reply:
It's exactly the same image, just renamed for clarity.

Reply:
oh, awesome. TY!